frontend/src/components/Navbar/Navbar.tsx

```diff
@@ -17,15 +18,18 @@ export const Navbar = (props: NavbarProps) => {
           alt="Government of B.C."
           src={Logo}
         />
-        {children}
+        <Typography variant="h4" component="div" sx={{ flexGrow: 1 }}>
+          {title}
+        </Typography>
+        <AuthenticationButton />
```

Comment:
Can we conditionally render the login button to only show up in the Navbar if isAuthenticated is true? Something similar to this: 

```import { AppBar, Box, Toolbar, Typography } from '@mui/material';
import { AuthenticationButton } from '../buttons/AuthenticationButton';
import Logo from '../../assets/logo.png';
import { useAuthentication } from '../../hooks';

type NavbarProps = {
  title?: string;
};

const Navbar = (props: NavbarProps) => {
  const { title } = props;
  const { isAuthenticated } = useAuthentication();

  return (
    <AppBar position="static">
      <Toolbar>
        <Box
          component="img"
          sx={{ height: 100, mr: 2 }}
          alt="Government of B.C."
          src={Logo}
        />
        <Typography variant="h4" component="div" sx={{ flexGrow: 1 }}>
          {title}
        </Typography>
        {isAuthenticated ? <AuthenticationButton /> : <div />}
      </Toolbar>
      <Box role="presentation" sx={{ height: 5 }} bgcolor="#FCBA19" />
    </AppBar>
  );
};

Navbar.defaultProps = {
  title: '',
};

export default Navbar;```

Reply:
@dallascrichmond  The Authentication button will render Login if not authenticated, and Logout if authenticated.  Your code example will have no button at all if authenticated, is that the behavior you are looking for?

Reply:
Do you mean 'not' authenticated? As it currently stands, when the user is not authenticated, the modal is shown with the login button and the login button is also on the Navbar in the background. I don't think we want to have the login button on the Navbar as well at this point since they won't interact with it from the Navbar. The code above only renders the logout button on the Navbar when the user is authenticated. So basically, the logout button only ever gets rendered in the Navbar and the login button only gets rendered in the modal.